waspc/src/Wasp/Analyzer/ErrorMessage.hs

```diff
@@ -3,13 +3,11 @@ module Wasp.Analyzer.ErrorMessage
   )
 where
 
-import Data.List (intercalate)
 import Wasp.Util (indent)
 
 makeFullErrorMsg :: String -> [String] -> String
-makeFullErrorMsg errorMsg errorCtxMsgs
-  | null errorCtxMsgs = errorMsg
-  | otherwise = intercalate "\n\n" [errorMsg, concatErrorCtxMsgs errorCtxMsgs]
+makeFullErrorMsg errorMsg errorCtxMsgs =
+  errorMsg ++ (if null errorCtxMsgs then "" else concatErrorCtxMsgs errorCtxMsgs)
```

Comment:
Ha, you did this at the end! I hope you were feeling fine with it, certainly didn't have to do it because of me, as I said it was just an idea, I am ok with both.

Reply:
No worries! As you said, it's a pretty trivial thing so it really doesn't matter. And you sold it to me with the reasoning (besides, it also doesn't require an import.